apps/web/migrations/0006_added_status_field.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Please rename the migrations to `0006_add_status_field_in_contact_model.py`

Reply:
@RishabhJain2018 done.